homeassistant/components/openweathermap/__init__.py

```diff
@@ -87,6 +89,12 @@ async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             entry, data=new_data, version=CONFIG_FLOW_VERSION
         )
```

Comment:
It looks like if they still have version 1 the migration will now not work

Reply:
if in first migration `if version == 1` i set version to `2`, will `async_migrate_entry` run again, so i can leave current construction as it is?
or now i need to combine two migrations into one?

Reply:
Its only going to run once

https://github.com/home-assistant/core/blob/9f9493c5042252fcbc00199d40693095ebd291f5/homeassistant/config_entries.py#L848

Reply:
done